apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -43,8 +55,8 @@ class ReviewApplication extends React.Component {
 
 
   render() {
+    console.log('++ReviewApplic render', this.props.issue)
     const { issue } = this.props
-
     const request = issue.requestsData[this.state.requestIndex]
```

Comment:
This line throws an error when reviewing an application from the IssueDetail view